### ISSUE TITLE
As a user, I can xcall with USDC on Mainnet

### ISSUE THREAD

**Issue by alexwhte** [connext/nxtp#1372]
**Problem**
In order to test Mainnnet we will need at least 2 chains (Ethereum and Optimism) and 1 asset (USDC)

**Ideas to solve this**
Fill pool with 100k USDC per security phases in some way and test bridging as a dev

What else is needed offchain? 

**Acceptance Criteria**
- [x] We have some USDC liquidity 
- [x] Able to bridge between Ethereum and Optimism mainnet 
- [ ] bridging fast works and is tested 

**Out of scope**
UI work will be handled by Coinhippo

**Tasks to file**
- [x] Deploy Mainnet
- [ ] Couple hardhat tasks?
- [ ] SDK releases?
- [x] Add USDC liquidity to Connext router and AMM pools

**Comment by rhlsthrm** [comment]
Blocked by #1398
